rust/parquet/src/file/reader.rs

```diff
@@ -796,6 +807,23 @@ mod tests {
         assert!(reader.is_err());
     }
 
+    #[test]
+    fn test_file_reader_into_iter() -> Result<()> {
+        let path = get_test_path("alltypes_plain.parquet");
+        let mut vec = vec![path.clone(), path.clone()]
+            .iter()
+            .map(|p| SerializedFileReader::try_from(p.as_path()).unwrap())
+            .flat_map(|r| r.into_iter())
+            .flat_map(|r| r.get_int(0))
+            .collect::<Vec<_>>();
+
+        vec.sort();
+
+        assert_eq!(vec, vec![0, 0, 1, 1, 2, 2, 3, 3, 4, 4, 5, 5, 6, 6, 7, 7]);
```

Comment:
Is this behaviour intended? Intuitively, I would have expected to see `vec![0,1,2,3,4,5,6,7, 0,1,2,3,4,5,6,7]` reflecting that you're reading 2 files sequentially. Am I missing something?

Reply:
Just bc I'm sorting the vec.

The values in the parquet are not sorted.. 
but i can remove the sort and keep the natural order if you guys think it makes more sense..

Reply:
Oops, I actually missed the sort just before this. I think it's better not to sort, makes it visually clearer.